webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -29,13 +41,37 @@ export default {
   },
   data() {
     return {
+      displayedNotifications: [],
       notifications: [],
     }
   },
   props: {
     placement: { type: String },
   },
+  watch: {
+    notifications: {
+      immediate: true,
+      handler(lastQueriedNotifications) {
+        if (lastQueriedNotifications && lastQueriedNotifications.length > 0) {
+          lastQueriedNotifications.forEach(updatedListNotification => {
+            const sameNotification = this.displayedNotifications.find(oldListNotification => {
+              return this.equalNotification(oldListNotification, updatedListNotification)
+            })
+            if (!sameNotification) this.displayedNotifications.unshift(updatedListNotification)
+          })
+        }
+      },
+    },
+  },
+
   methods: {
+    async updateNotifications() {
+      try {
+        this.$apollo.queries.notifications.refetch()
+      } catch (err) {
+        throw new Error(err)
+      }
+    },
```

Comment:
I would prefer to have this callback deleted. I believe it creates bad user experience, see above :point_up:

Reply:
Why bad user experience?
Because of the possible error?
So I shall use `this.$apollo.queries.notifications.refetch()` directly?
Or get rid of the update at all? Like you said …

I don’t agree with the last, like I said above …

Reply:
Bad user experience because of the situation where you are about to click on a notification but in the very moment when you want to click on it, the list gets updated and you suddenly click on sth. else.

Reply:
I implemented now two new things and got rid of the click update. See my general comment of the changes 👇🏼